### ISSUE TITLE
Consider adding a note to the provider README regarding riverpod

### ISSUE THREAD

**Issue by AKushWarrior** [rrousselGit/river_pod#36]
**Describe the solution you'd like**
Hi there! You noted that this package would need to be tested and approved by the larger Dart community before it could be considered for merging as a breaking version of provider. Given that two versions of riverpod (hooks and flutter) have each racked up a 90+ rating in less than a month, I think it's fair to say that there is a demand for this package and that the problems it solves are real.

On the other hand, it's clearly far too early for a merge to provider. Seeing this, I think that a smart decision would be to add a small note to the top of the provider README, in the form of:

"Check out my experimental rewrite of provider, riverpod! It attempts to solve the problems that are architecturally embedded in this package, and provides a cleaner interface to achieve the same goals." (where the word riverpod contains a hyperlink to https://riverpod.dev)

I think that this might get you some more testers and users to help riverpod achieve maturity, while still conveying that riverpod isn't currently stable.

**Comment by rrousselGit** [comment]
Good idea. I'll do that when I have finished my article introducing Riverpod, since I have yet to officially announce it.
